activerecord/lib/active_record/validations.rb

```diff
@@ -62,6 +62,8 @@ def save!(**options)
     #
     # If the argument is +false+ (default is +nil+), the context is set to <tt>:create</tt> if
     # {new_record?}[rdoc-ref:Persistence#new_record?] is +true+, and to <tt>:update</tt> if it is not.
+    # If the argument is an array of contexts, +post.valid?([:create, :update])+, then multiple contexts
+    # are validated.
```

Comment:
The contexts aren't validated.
The validations are run within contexts.
```suggestion
    # If the argument is an array of contexts, +post.valid?([:create, :update])+, the validations  are run
    # within multiple contexts.
```